src/base.py

```diff
@@ -0,0 +1,48 @@
+import enum
+
+
+class WithRequester(object):
+
+    API_URL = NotImplementedError
+
+    def __init__(self, requester):
+        self._requester = requester
```

Comment:
Do we have a reason to have `_requester` private, and a `requester` property? Why not to set the requester's `API_URL` here, making the requester public, and get rid of the property?

Reply:
For now - yes, because requester is a single object shared between all the APIs and if we set `API_URL` for it in constructor - it will be `API_URL` of the latest created class, working with api. It should be refactored, but for now I made a fix for it, using property and setting requester's `API_URL` each time we call it. Noticed it in #8